src/viam/components/camera/camera.py

```diff
@@ -48,7 +48,10 @@ async def get_image(
 
         ::
 
-            my_camera = Camera.from_robot(robot=robot, name="my_camera")
+           # from viam.media.video import CameraMimeType    
+           # Add the above to imports at top of file!
```

Comment:
They can add it anywhere above where it's used. If they use python, they don't need to be told the rest
```suggestion
           from viam.media.video import CameraMimeType
```

Reply:
Okay I agree, I was just doing this because that's how we do it in another docs-side example. I'll change that one too